lib/counter_culture/counter.rb

```diff
@@ -14,6 +14,7 @@ def initialize(model, relation, options)
       @foreign_key_values = options[:foreign_key_values]
       @touch = options.fetch(:touch, false)
       @delta_magnitude = options[:delta_magnitude] || 1
+      @execute_after_commit = options.fetch(:execute_after_commit, false)
```

Comment:
@kulte, should this be `true` by default when not set, as it's the current behaviour?

I agree, this would be a useful option.